GitHub-case-study.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Example Product Case Study
+keywords: development platforms, github
+tags: Development Platforms, github
+sidebar: development-platforms
+permalink: development-platforms/case-study-github.html
+folder: development-platforms
+---
+
+# <p align="center"><span>GitHub Case Study<span></p>
+
+<p align="center"><img src="Images/github.png"><p>
+
+## Introduction
+
+GitHub is an Internet hosting service for software development and version control using Git. It is a web based platform where users can store their code and collaborate together on a coding project. GitHub has a lot of great tools/features like github desktop, actions, project management, teams, etc.
+
+## Brief History
+
+GitHub was founded in 2008 by Tom Preston-Werner, Chris Wanstrath, P.J. Hyett, and Scott Chacon. GitHub has had an enormous growth since it was founded and today it is one of the most popular platforms for web hosting and code sharing. GitHub's headquarters is based in San Francisco in the state of California, in the united states of america. The CEO of GitHub is named Thomas Dohmke. GitHub currently has about 2500 employees and in 2018 it had about 200-300 million USD of revenue.
+
+### From the beginning to the present time (2023)
+
+**github blooming**
+
+- (year 2010) After only two years since it's launch github was hosting over 1 million repositories witch doubled a year later.
+- (year 2011) In the period of January to May 2011 ReadWriteWeb reported that GitHub had surpassed SourceForge and Google Code in total number of commits.
+- (year 2013) It was hosting 5 million repositories and had over 3 million users. Before this year ended the amount of repositories had doubled again and it was now hosting over 10 million repositories.
+
```

Comment:
These sentences are a tad convoluted, and there are some spelling mistakes present. Perhaps try rephrasing them so they read more easily?
For example `After only two years since it's launch github was hosting over 1 million repositories witch doubled a year later.` could become `Two years after its launch, GitHub was hosting over 1 million repositories. A year later, this had doubled.`